Content.Server/Shuttles/Systems/EmergencyShuttleSystem.cs

```diff
@@ -206,6 +268,24 @@ public void CallEmergencyShuttle(EntityUid stationUid, StationEmergencyShuttleCo
                 _chatSystem.DispatchStationAnnouncement(stationUid, Loc.GetString("emergency-shuttle-docked", ("time", $"{_consoleAccumulator:0}"), ("direction", angle.GetDir())), playDefaultSound: false);
             }
 
+            // shuttle timers
+            var time = TimeSpan.FromSeconds(_consoleAccumulator);
+            if (TryComp<DeviceNetworkComponent>(stationShuttle.EmergencyShuttle.Value, out var netComp))
+            {
+                AllEntityQuery<StationCentcommComponent>().MoveNext(out var centcomm);
```

Comment:
Just dump this on a method instead IMO, like "GetCentcomm" or something so at least we only have 1 spot to update if we have to change it.